man/has_name.Rd

```diff
@@ -2,15 +2,7 @@
 % Please edit documentation in R/has-name.R
 \name{has_name}
 \alias{has_name}
-\title{Convenience function to check presence of a named element}
-\usage{
```

Comment:
It's a pity `\usage` gets lost. Arguments could be brought in with `@inheritParams`.

Anyway, I've seen many packages in the tidyverse reexport without providing redundant documentation. I'd suggest to use this pattern here as well, and figure out a better way later; this may well mean that we need support from R core.